src/cork/client.rs

```diff
@@ -34,6 +82,20 @@ impl CorkQueryClient {
         id: &str,
     ) -> Result<Response<QueryScheduledCorksByIdResponse>, Status> {
         let request = QueryScheduledCorksByIdRequest { id: id.to_string() };
-        self.inner.query_scheduled_corks_by_id(request).await
+
+        self.inner_cork.query_scheduled_corks_by_id(request).await
+    }
+
+    pub async fn get_axelar_scheduled_corks_by_id(
```

Comment:
I saw this and was wondering what called it -- it turns out that I think this PR doesn't included the proposal thread looking at Axelar gov props.

Reply:
ah. I can do it in another PR